iceoryx_hoofs/test/moduletests/test_posix_thread.cpp

```diff
@@ -32,19 +32,11 @@ using namespace iox::units::duration_literals;
 class Thread_test : public Test
 {
   public:
-    Thread_test()
+    void SetUp() override
```

Comment:
I think we should remove both `SetUp` and `TearDown` as the base implementations do nothing as well.